lib/Conversion/TritonGPUToLLVM/Utility.cpp

```diff
@@ -419,7 +420,10 @@ void storeDistributedToShared(MemDescType dstTy, RankedTensorType srcTy,
         store(vec, vecAddr)
             .setAlignment(vecTy.getNumElements() *
                           elemLlvmTy.getIntOrFloatBitWidth() / 8);
+        ++((*llvmOpCount).first);
```

Comment:
Need to check `llvmOpCount` is not `nullptr` first.

Reply:
Done